website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -288,6 +288,14 @@ Once this is done, there are two ways to run the operator:
 - As Go program outside a cluster
 - As a Deployment inside a Kubernetes cluster
 
+### Configuring your test environment
+
+## Configuring your test environment
```

Comment:
```suggestion
```

Reply:
If the users do not follow up it then they will face an issue in the face in the next step. So, IMO keep the topic make clear that is more one step that requires to get done. Also, make clear the separation of the topics. However, it is very important we clean up this doc. It is not a quick start. My personal goal is to do a PR with that as a follow-up. For now, I just would like to solve the issue that is a blocker for 1.0.0. 



 We need to improve the full doc it is not a quick start btw .. So, let's keep the

Reply:
Github is hiding the next line up. The problem is that this title was added twice.

Reply:
ohhh .. I will check tks